src/http/url.cc

```diff
@@ -67,8 +67,7 @@ inline char char_to_hex(unsigned char val) {
     return "0123456789ABCDEF"[val];
 }
 
-template<bool ReplacePlus>
-bool decode(const std::string_view& in, sstring& out) {
+bool decode(bool replace_plus, const std::string_view& in, sstring& out) {
```

Comment:
This function was made a template in this PR, so either keep it as template, or make it accept `replace_plus` boolean argument in the 1st patch (the one that made it a template)